app/javascript/dashboard/routes/dashboard/settings/components/BaseSettingsList.vue

```diff
@@ -0,0 +1,6 @@
+<script setup></script>
+<template>
+  <div class="flex flex-col w-full h-full gap-3">
+    <slot />
+  </div>
+</template>
```

Comment:
Can this not be encapuslated in a Layout level component itself? 

SettingsLayout.vue
```vue
<template>
    <div>
        <slot for header>
        <slot for body>
    </div>
</template>
```